package.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "cso-api-next",
+  "name": "cso-production",
```

Comment:
I don't know if this name has any bearing anywhere, but how about calling it "cso-api-production" rather than "cso-production"? It's more specific.

Reply:
The name of the package doesn't have any bearing on the end result, but it doesn't hurt to call it something more straightforward for sure.